code/game/objects/obj_defense.dm

```diff
@@ -95,8 +95,14 @@
 
 /obj/attack_slime(mob/living/simple_animal/slime/user, list/modifiers)
 	if(!user.is_adult)
+		if(user.client)
+			log_combat(user, src, "attempted to attack")
 		return
-	attack_generic(user, rand(10, 15), BRUTE, MELEE, 1)
+	if(user.client)
```

Comment:
NPC slimes use attack_slime() on objects that bump them.

Reply:
Soooo, what should i do then?

Reply:
Honestly I don't see a need to be checking for a client at all, just log the attack normally.

Reply:
I think that there is no reason for logging NPC attacks, but i can make it log even without a client